Sources/MosaiqueAssetsPicker/Views/AssetDetailCell.swift

```diff
@@ -94,7 +94,7 @@ final class AssetDetailCell: UICollectionViewCell, AssetDetailCellBindable {
     
     public func updateSelection(isItemSelected: Bool) {
         if isItemSelected {
-            imageView.layer.borderColor = configuration.selectionColor.cgColor
+            imageView.layer.borderColor = configuration?.selectionColor.cgColor
```

Comment:
I can understand it fixes that crash, however, it seems that just suppress runtime-error.
With this, we will miss silently error.
`configuration` seems it's declared as guaranteed non-nill  by developer

So the actual root cause might be outside this cell.